src/providers/python.rs

```diff
@@ -171,6 +176,20 @@ impl PythonProvider {
 
             install_phase.add_cache_directory(PIP_CACHE_DIR.to_string());
 
+            return Ok(Some(install_phase));
+        } else if app.includes_file("Pipfile") {
+            let cmd = if app.includes_file("Pipfile.lock") {
+                "PIPENV_VENV_IN_PROJECT=1 pipenv install --deploy"
```

Comment:
What does `PIPENV_VENV_IN_PROJECT` do?

Reply:
by default Pipenv creates an environment directory in some random location (for example `/root/.local/share/virtualenvs/app-4PlAip0Q`).

`PIPENV_VENV_IN_PROJECT` tells it that there is an already activated `venv` environment, So Pipenv will use the same directory instead of creating new one (in our case it's ` /app/.venv`)